src/index.js

```diff
@@ -533,17 +533,45 @@ export default class RNPickerSelect extends PureComponent {
         );
     }
 
+    renderWeb() {
+        const { disabled, Icon, style, pickerProps } = this.props;
+        const { selectedItem } = this.state;
+
+        return (
+            <View style={[defaultStyles.viewContainer, style.viewContainer]}>
+                <Picker
+                    style={[
+                        Icon ? { backgroundColor: 'transparent' } : {}, // to hide native icon
+                        style.inputWeb,
+                        this.getPlaceholderStyle(),
```

Comment:
does this work on web?

Reply:
Placeholder style does not, I'll remove, if we keep as a separate component